apis/rust/node/src/node/mod.rs

```diff
@@ -43,6 +47,31 @@ pub struct DoraNode {
     cache: VecDeque<ShmemHandle>,
 
     dataflow_descriptor: Descriptor,
+    detached: bool,
+}
+
+fn get_info(
+    session: &mut TcpRequestReplyConnection,
+    dataflow_id: String,
+    node_id: NodeId,
+) -> Result<NodeConfig, eyre::ErrReport> {
+    let reply_raw = session
+        .request(
+            &serde_json::to_vec(&ControlRequest::NodeConfig {
+                dataflow_id,
+                node_id,
+            })
+            .unwrap(),
+        )
+        .wrap_err("failed to send list message")?;
+    let reply: ControlRequestReply =
+        serde_json::from_slice(&reply_raw).wrap_err("failed to parse reply")?;
+    let node_config = match reply {
+        ControlRequestReply::NodeConfig { node_config } => node_config,
+        ControlRequestReply::Error(err) => bail!("{err}"),
+        other => bail!("unexpected list dataflow reply: {other:?}"),
+    };
+    Ok(node_config)
 }
```

Comment:
This should go to the daemon, not to the coordinator.

Reply:
Done!